custom/icds_reports/tasks.py

```diff
@@ -1679,3 +1680,17 @@ def _get_primary_data_for_cases(db, domain, day, case_type):
         type=case_type
     )
     return matching_cases.values_list('case_id', 'type', 'server_modified_on')
+
+
+@periodic_task_on_envs(
+    settings.ICDS_ENVS,
+    run_every=crontab(minute=30, hour=12),  # To on 6PM IST
+    acks_late=True,
+    queue='icds_aggregation_queue'
+)
+def update_dashboard_activity_report(target_date=None):
```

Comment:
hey even if this is not part of standard aggregation?

Reply:
I'm not saying to add this to the current aggregation task, but creating a new task/DAG for this. I think it will be easier to maintain long term if there's only one place (airflow) where such queries are run